src/chrome/komodo/locale/en-US/pref/pref.dtd

```diff
@@ -708,6 +708,9 @@ adding an additional caret at the clicked position.">
 <!ENTITY javascriptFolder.description "Specify any directories that you want Komodo to use for autocomplete and calltips. Komodo will recursively scan these directories for information.">
 <!ENTITY javascriptExcludeFolder.description "Specify any directories you want to exclude from the scan.">
 <!ENTITY javascriptNamespaceMapping.description "Specify any custom namespace mappings for Komodo to use when looking up modules in 'require' statements for code intelligence. These mappings are useful for projects that have non-standard module locations.">
+<!ENTITY eslintChecking.caption "EsLint Linter Checking">
+<!ENTITY eslintChecking.checkbox "Enable the eslint linter">
+<!ENTITY eslintChecking.description "Use the eslint linter when doing background syntax checking.">
```

Comment:
They write it as "ESLint", please use caps consistently and the same as the authors.